layouts/shortcodes/notices/premiumShip.html

```diff
@@ -0,0 +1 @@
+{{ partial "notice.html" (merge (dict "page" .Page) (dict "noticeTitle" "Premium Ship" "noticeDescription" "This ship is a premium ship and is only obtainable by purchasing it from the Robux shop menu." "noticeColor" "linear-gradient(270deg, #FF4600, #FFD966)" ) ) }}
```

Comment:
Gradients don't work in the same way as they did in Mediawiki. You have to use a Tailwind CSS class. I recommend setting this to a flat color instead. We *can* do gradients, but I think making the required changes to the notice partial is out of scope for this PR.